<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GaussianBlurForm</class>
 <widget class="QWidget" name="GaussianBlurForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>172</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Size</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>w</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_size_w">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>h</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_size_h">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Sigma</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="sigmaXSpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="sigmaYSpinBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Border</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_border">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <item>
        <property name="text">
         <string>Constant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Replicate</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Reflect</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Reflect 101</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Transparent</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Time ms:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_time">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
